client/starwhale/base/bundle_copy.py

```diff
@@ -55,6 +56,23 @@ class _UploadPhase:
     CANCEL = "CANCEL"
 
 
+class FileType(Enum):
```

Comment:
add `@unique` for FileType?

`from enum import unique`

Reply:
done